### ISSUE TITLE
Unintelligible text [DOC]

### ISSUE THREAD

**Issue by smacrakis** [opensearch-project/documentation-website#7689]
**What do you want to do?**
 
- [ X] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request.

What does "Example 3: Filter and sort be casted parsed field" mean? The example seems to show filtering on a parsed field and then sorting on that same field. Maybe the title should be "Example 3: Filter and sort a parsed field".

The examples all work with a sample data set "accounts". But that doesn't seem to be in the standard distribution. Where can I find it so that I can try out the examples?

Also, I can't find the documentation for the function ''num'' in "sort num(streetNumber)". I'l guess that it converts a string to a number (very handy) -- but where is it documented? I couldn't get it to work in OpenSearch, either.

***Version:** List the OpenSearch version to which this issue applies, e.g. 2.14, 2.12--2.14, or all.

2.15
 
**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

**Comment by hdhalter** [comment]
Thanks, @smacrakis. Can you please add some links to the applicable content?

**Comment by smacrakis** [comment]
Done. Sorry about that. I wonder if we could get the "create an issue" link to pre-populate the title with the URL for lazy/careless people like me.

**Comment by hdhalter** [comment]
> Done. Sorry about that. I wonder if we could get the "create an issue" link to pre-populate the title with the URL for lazy/careless people like me.

It's a good idea, but we prefer that people click "Edit this page" and make the change to the page directly. :) Even if you create a PR without all the answers, we can find an SME that can review and supplement the added content.

**Comment by smacrakis** [comment]
So you're saying that I should write something like:

You can find the ''accounts'' test data at XXX. (where I don't know what XXX is)

The "sort" command supports the syntax sort num(field) to interpret the field as numerical.  <<< is this true?? because num(...) is not supported elsewhere

**Comment by hdhalter** [comment]
> So you're saying that I should write something like:
> 
> You can find the ''accounts'' test data at XXX. (where I don't know what XXX is)
> 
> The "sort" command supports the syntax sort num(field) to interpret the field as numerical. <<< is this true?? because num(...) is not supported elsewhere

Yes! Collaborating on a pull request is a good way to fast-track a fix.

**Comment by smacrakis** [comment]
I have no expertise in this area.

**Comment by kolchfa-aws** [comment]
The PPL documentation has been completely rewritten, so this issue is no longer relevant.
